Jenkinsfile

```diff
@@ -0,0 +1,52 @@
+try { // massive try{} catch{} around the entire build for failure notifications
+    timestamps {
+        timeout(time: 60, unit: 'MINUTES') {
+            node('fedora-28') {
+                cleanWs()
+
+                stage('Prepare ENV') {
+                    sh """
+                        sudo curl --insecure -o /etc/pki/ca-trust/source/anchors/RH-IT-Root-CA.pem \
+                             https://password.corp.redhat.com/RH-IT-Root-CA.crt && sudo update-ca-trust extract
+
+                        sudo curl -o /etc/yum.repos.d/rcm-tools-fedora.repo \
+                             http://download.devel.redhat.com/rel-eng/RCMTOOLS/rcm-tools-fedora.repo
+
+                        sudo dnf -y install git krb5-workstation python2 python2-pip \
+                             python-gobject libmodulemd python2-pygit2 python2-yamlordereddictloader
+
+                        git clone -b spade_jenkinsfile https://github.com/yashvardhannanavati/Spade.git
```

Comment:
Should we move this to the `release-engineering` org?  (I had trouble finding Spade in @sarah256's account the first time I went looking...)

Reply:
Yeah, I did talk to alex about this on IRC. I will move it to the `release-engineering` org once it's good to go.